### DISCUSSION TITLE
theming -fontFamily variables

### DISCUSSION THREAD

**Discussion by akaflipper** [tailwindlabs/tailwindcss#4137] [Help]
Hi, 
following @simonswiss fantastic video series on YouTube and trying to extend the theming with CSS Variables episode - [https://www.youtube.com/watch?v=MAtaT8BZEAo&t=942s](url) - to cover fontFamily theming ... the bg, text colours & opacity work great .. but I can't get fontFamily to 'variablise'. Not sure if its possible but here's what i've got:

[https://play.tailwindcss.com/Umm8NjbKaL](url)

/resources/css/app.css - creating variables for font styles eg: --font-sans: ['MaisonNeue-Book','Arial','sans-serif']; etc

<img width="711" alt="Screenshot 2021-04-21 at 09 04 13" src="https://user-images.githubusercontent.com/66594059/115513243-216ae080-a283-11eb-8706-1073930f3ee3.png">

tailwind.config.js - referencing them within a theme->skin 
<img width="554" alt="Screenshot 2021-04-21 at 09 04 06" src="https://user-images.githubusercontent.com/66594059/115511517-35154780-a281-11eb-8ebf-a76374c6fe05.png">

test blade - calling font-skin-display and font-skin-display
<img width="628" alt="Screenshot 2021-04-21 at 09 04 49" src="https://user-images.githubusercontent.com/66594059/115511573-43636380-a281-11eb-876d-7765cdaa1b1a.png">

No fonts appear to be imported/referenced within generated /public/css/app.css file and i think that's my problem ... 

 I've tried a bunch of different ways to define the variables and reference them - --font-serif, in double quotes to escape chars etc ... but no joy. 
 
thanks a bunch!
g

**Comment by simonswiss** [comment]
Hey!

Unfortunately, the `fontFamily` key in Tailwind doesn't support nested objects (like `fontFamily: { skin: {} }`) like colors do in my video.

You can still use the `skin` prefix, but you'd need to define it like so:

```js
theme: {
  extend: {
    fontFamily: {
      'skin-serif': 'var(--font-serif)'
    }
  }
}
```

Also, since the config file will literally point to the CSS variable (instead of parsing a JS array), you should define your font families with the CSS syntax instead, in your CSS file:

```css
:root {
  --font-serif: 'MaisonNeue-Book', arial, sans-serif;
}
```

That _should_ work! 👍

**Reply by akaflipper** [reply]
Awesome thanks @simonswiss .. got it working! very cool! appreciate the help!
